xml/admin_about.xml

```diff
@@ -266,7 +263,10 @@
        The &ceph; manager (MGR) collects the state information from the whole
```

Comment:
I think there is an entity for this? &mgr;

Reply:
+1